R/image-interact.R

```diff
@@ -93,10 +93,10 @@ brushedPoints <- function(df, brush, xvar = NULL, yvar = NULL,
   use_y <- grepl("y", brush$direction)
 
   # Try to extract vars from brush object
-  xvar      <- xvar      %OR% brush$mapping$x
-  yvar      <- yvar      %OR% brush$mapping$y
-  panelvar1 <- panelvar1 %OR% brush$mapping$panelvar1
-  panelvar2 <- panelvar2 %OR% brush$mapping$panelvar2
+  xvar      <- xvar      %||% brush$mapping$x
+  yvar      <- yvar      %||% brush$mapping$y
+  panelvar1 <- panelvar1 %||% brush$mapping$panelvar1
+  panelvar2 <- panelvar2 %||% brush$mapping$panelvar2
```

Comment:
I wouldn't be that surprised if there are some user apps out there that have mistakenly passed `NA` and are relying it falling through to `brush$mapping`. That being said, I want to live in a `%||%` world, so I think it'd be worth the short-term pain

Reply:
Yeah, if anyone is doing that, I think they should fix it. I suppose we could warn in that case, though.